### PR TITLE
Ticket #8 solved

### LINKED ISSUE #8: Get list of products over a specified price
When the client performs a GET request to the products resource, and specifies a minimum price as a query string parameter, the results should contain products whose `price` property value is equal to, or greater than, the value of the query string parameter.

#### Example

http://localhost:8000/products?min_price=150

### PR DESCRIPTION
**Pull request by Kamiran79**
Description of PR that completes issue here...

## Changes

- Item 1: add new line 251 in views/product.py the line to pull min_price from the get request
- Item 2: add new lines 262 to 267 in views/product.py and that applying the filter for product price

## Requests / Responses

If this PR contains code that defines a new request/response, or changes an existing one, please put the JSON representations here.

**Request**

GET `/products?min_price=1950`  Getting products that price => min_price

**Response**

"GET /products?min_price=1913 HTTP/1.1" 200  OK

```json
[
    {
        "id": 41,
        "name": "Impreza",
        "price": 1913.81,
        "number_sold": 0,
        "description": "1998 Subaru",
        "quantity": 4,
        "created_date": "2019-03-15",
        "location": "Dingjiaqiao",
        "image_path": null,
        "average_rating": -1
    },
    {
        "id": 60,
        "name": "Elantra",
        "price": 1989.84,
        "number_sold": 0,
        "description": "1994 Hyundai",
        "quantity": 1,
        "created_date": "2019-01-19",
        "location": "Sosnovoborsk",
        "image_path": null,
        "average_rating": -1
    }
]
```

## Testing

Description of how to test code...

- [ ] Run  Get http://localhost:8000/products?min_price=1913 was return all products
- [ ] Know that after the first run the filter not applied.
- [ ] Add the filter on the views/product.py and that was on line 251, and 262-267
- [ ] Run the Get request again, got price filter base on minimum price.
- [ ] Run more get to testing the filter bringing the correct filter.

## Related Issues

- Fixes #8

### CODE REVIEW

File: bangazonapi/views/product.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from rest_framework.response import Response
 from rest_framework import serializers
 from rest_framework import status
-from bangazonapi.models import Product, Customer, ProductCategory
+from bangazonapi.models import Product, Customer, ProductCategory, product
```
Comment (mitchellblom) [inline bangazonapi/views/product.py]: Why are there two `Product` and `product` models?

**Review by mitchellblom — CHANGES_REQUESTED** [review]

File: bangazonapi/views/product.py (hunk 1 above)
Comment (Kamiran79) [reply bangazonapi/views/product.py]: it is not used .. didn't pay attention to that I will delete it and test then push the update.

**Review by mitchellblom — CHANGES_REQUESTED** [review]
If this is a different ticket (#11) please PR it separately from #8.

**Review by Kamiran79 — COMMENTED** [review]
updated as requested.

File: bangazonapi/views/product.py [hunk 2]
```diff
@@ -248,6 +248,7 @@ def list(self, request):
         order = self.request.query_params.get('order_by', None)
         direction = self.request.query_params.get('direction', None)
         number_sold = self.request.query_params.get('number_sold', None)
+        price = self.request.query_params.get('min_price', None)
```
Comment (mitchellblom) [inline bangazonapi/views/product.py]: `price` suggests any price. Suggest that you specify minimum somehow in the variable.

**Review by mitchellblom — CHANGES_REQUESTED** [review]
This solution is not for this ticket. I believe you meant to put #9 in your PR template instead of #8. Could you please fix that?

**Review by mitchellblom — APPROVED** [review]
Thank you

File: bangazonapi/views/product.py (hunk 2 above)
Comment (Kamiran79) [reply bangazonapi/views/product.py]: Update the variable price to min_price

**Comment by Kamiran79** [comment]
There was two thinks changes the price variable changed to min_price that to not make issue on line 266 when appling the filter.

**Pull request merged**